lighthouse-core/audits/seo/hreflang.js

```diff
@@ -19,6 +19,10 @@ const UIStrings = {
   description: 'hreflang links tell search engines what version of a page they should ' +
     'list in search results for a given language or region. [Learn more]' +
     '(https://web.dev/hreflang/).',
+  /** A failure reason for the `hreflang` Lighthouse audit. This failure reason is shown when the hreflang language code is unexpected. */
```

Comment:
cc @exterkamp is this enough context for a translator? I imagine our explanations might need to be self-contained rather than rely on knowledge of what "`hreflang` Lighthouse audit" means.

maybe?

```suggestion
  /** A failure reason for a Lighthouse audit that flags incorrect use of the `hreflang` attribute on `link` elements. This failure reason is shown when the hreflang language code is unexpected. */
```

same goes for the next description